tutorials/connect-jamf-pro-to-smallstep.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Connect Jamf Pro to Smallstep
+html_title: Connect Jamf Pro to Smallstep
+description: Configure Jamf Pro to deploy the Smallstep Agent and distribute certificates and configuration to Mac clients.
+---
+
+Smallstep can integrate with Jamf Pro to synchronize your device inventory, and enroll your fleet with Smallstep using the Smallstep Agent. In this document, we will configure your Jamf Pro instance for use with your Smallstep team.
+
+# Prerequisites
+
+You will need:
+
+- A [Smallstep team](https://smallstep.com/signup)
+- A [Jamf Pro](https://www.jamf.com/products/jamf-pro/) tenant
+- A [package distribution point](https://docs.jamf.com/10.35.0/jamf-pro/install-guide-linux/About_Distribution_Points.html) configured in Jamf Pro
+
+# Step-by-step instructions
+
+### 1. Create API Client in Jamf Pro
+
+This API client will allow Smallstep to read your Jamf device inventory and manage the webhooks necessary for ongoing inventory syncing.
+
+1. In Jamf Pro, choose ⚙️ **Settings**
+2. Under the **System** tab, choose **API Roles and Clients**
+3. Add an **API Role**
+    
+    Display name: Smallstep
+    
+    For Privileges, choose:
+    
+    - Read Mobile Devices
+    - Read Computers
+    - Webhooks: Create, Delete, Read, Update
```

Comment:
This is currently not required, as we're not creating webhooks programmatically (at least not yet, afaik). I guess it's fine for future use cases, and so users don't need to update it.